keyboards/crkbd/rev1/rev1.h

```diff
@@ -1,13 +1,17 @@
 #pragma once
 
-#include "../crkbd.h"
+#include "crkbd.h"
 
 //void promicro_bootloader_jmp(bool program);
 #include "quantum.h"
 
-#ifdef RGBLIGHT_ENABLE
-//rgb led driver
-#include "ws2812.h"
+#ifdef PROTOCOL_LUFA
+  #include "lufa.h"
+  #include "split_util.h"
+#endif
+
+#ifdef SSD1306OLED
+  #include "ssd1306.h"
```

Comment:
This should probably use 4 space indentation. Same goes for the rest of the file (lines 9–10, 27–41, 44–54).

```suggestion
    #include "ssd1306.h"
```

Reply:
Fixed.  That better?

Reply:
Yep, that's better. I see you opted to keep the 2-space indentation in the layout macros, why is that?

Reply:
Because I switched to macOS, and for whatever reason, it wants to ignore clang format setting, apparently....